ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java

```diff
@@ -276,4 +294,47 @@ public APIGatewayProxyResponseEvent handleRequest(
                             }
                         });
     }
+
+    private void queueSPOTRequest(
+            String sessionId,
+            String sectorIdentifier,
+            UserProfile userProfile,
+            Subject pairwiseSubject,
+            UserInfo userIdentityUserInfo)
+            throws JsonProcessingException {
+
+        var spotClaimsBuilder =
+                SPOTClaims.builder()
+                        .withVot(userIdentityUserInfo.getClaim(VOT.getValue()).toString())
+                        .withVtm(
+                                buildURI(
+                                                configurationService
+                                                        .getOidcApiBaseURL()
+                                                        .orElseThrow(),
+                                                "/trustmark")
+                                        .toString());
+
+        var ipvCredentials = userIdentityUserInfo.getClaim(IPV_CREDENTIALS.getValue());
+        if (ipvCredentials != null && ipvCredentials instanceof ArrayList) {
+            spotClaimsBuilder.withCredentials(
+                    (ArrayList) userIdentityUserInfo.getClaim(IPV_CREDENTIALS.getValue()));
+        }
+
+        var ipvCoreIdentity = userIdentityUserInfo.getClaim(IPV_CORE_IDENTITY.getValue());
+        if (ipvCoreIdentity != null) {
+            spotClaimsBuilder.withIdentityCredentials(
+                    userIdentityUserInfo.getClaim(IPV_CORE_IDENTITY.getValue()));
+        }
```

Comment:
This looks good for now. We will need to re-visit this when IPV return all the info as we may not want to send anything to SPOT if credentials are missing from the IPV user-identity response. Maybe add a placeholder jira ticket in slice 2?

Reply:
now copying everything across except sub as spot has a separate field for that 'out_sub'.